components/about/contactForm.js

```diff
@@ -61,6 +61,7 @@ const ContactForm = () => {
       await sendSupportFeedback({ name, email, topic, body })
       setFormError(false)
       setSent(true)
+      contactFormSubmitted()
```

Comment:
at some point you reeeeeally need to add prop types to these components

Reply:
yea we do on most components it was just an oversight here that i forgot to add it